docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -346,7 +393,133 @@ txn_type      | debit
+
+However, if we instead connect to a node in eu-central-1 and run the exact same transaction:
+
+```sql
+BEGIN;
+```
+```
+BEGIN
+```
+```sql
+INSERT INTO transactions VALUES (100, 10002, 'EU', 'checking', 400.0, 'debit');
+```
+```
+INSERT 1 0
+```
+```sql
+INSERT INTO transactions VALUES (100, 10003, 'EU', 'checking', 400.0, 'credit');
+```
+```
+INSERT 1 0
+```
+```sql
+COMMIT;
+```
+```
+COMMIT
+```
+
+we are now able to finish and commit the transaction without error.
+
+Sometimes though, we might want to run a transaction that writes data to multiple regions, for example:
+
+```sql
+BEGIN;
+INSERT INTO transactions VALUES (100, 10004, 'US', 'checking', 400.00, 'debit');
+INSERT INTO transactions VALUES (200, 10005, 'EU', 'checking', 400.00, 'credit');
+COMMIT;
+```
+
+Running this transaction will fail whether we run it from us-west-2 or eu-central-1. The solution is to mark the transaction as a global transaction as follows:
+```sql
+SET force_global_transaction = TRUE;
+```
+```
+SET
+```
+```sql
+BEGIN;
+```
+```
+BEGIN
+```
+```sql
+INSERT INTO transactions VALUES (100, 10004, 'US', 'checking', 400.00, 'debit');
+```
+```
+INSERT 1 0
+```
+```sql
+INSERT INTO transactions VALUES (200, 10005, 'EU', 'checking', 400.00, 'credit');
+```
+```
+INSERT 1 0
+```
+```sql
+COMMIT;
+```
+```
+COMMIT
+```
+
+Setting `force_global_transaction = TRUE` tells YugabyteDB to use the `system.transactions` transaction table instead, which is presumed to be globally replicated, and lets us run distributed transactions that span multiple regions.
+
+{{< note title="Note" >}}
+While all distributed transactions can run without problems under `force_global_transaction = TRUE`, such transactions may incur significantly higher latencies when committing the transaction, since YugabyteDB must perform consensus across multiple regions to write to `system.transactions`. This should only be used when necessary, and `force_global_transaction = FALSE` (the default setting) should be used whenever possible.
+{{< /note >}}
+
+Finally, let's say we want to delete the very last row we just inserted. If we run the following query connected to eu-central-1 as a local transaction:
+
+```sql
+SET force_global_transaction = FALSE;
+```
+```sql
+DELETE FROM transactions WHERE user_id = 200 AND account_id = 10005;
+```
+```
+ERROR:  Illegal state: Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64: . Errors from tablet servers: [Illegal state (yb/client/transaction.cc:284): Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64]
+```
+
+the query will once again error out. This is because we are deleting from the main table (`transactions` rather than `transactions_eu_west_1`) and not specifying the partition column (there's no `geo_partition = 'EU'` clause), which means that YugabyteDB is unable to tell that the row being deleted is in fact located in eu-central-1. To fix this, we could instead run:
```

Comment:
````suggestion
Finally, let's say we want to delete the row we just inserted. If we run the following query connected to eu-central-1 as a local transaction, the query once again errors out:

```sql
SET force_global_transaction = FALSE;
DELETE FROM transactions WHERE user_id = 200 AND account_id = 10005;
```

```output
ERROR:  Illegal state: Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64: . Errors from tablet servers: [Illegal state (yb/client/transaction.cc:284): Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64]
```

We are attempting to delete from the main table (`transactions` rather than `transactions_eu_west_1`) and not specifying the partition column (there's no `geo_partition = 'EU'` clause). This means that YugabyteDB is unable to tell that the row being deleted is in fact located in eu-central-1. To fix this, we could instead run:
````